docs/application/native/guides/device/resource-monitor.md

```diff
@@ -0,0 +1,305 @@
+# Resource Monitor
+
+You can monitor various resources in a unified API with [resource type](#resource_types), [attributes](#resource_attributes), and some [controls](#resource_controls).
+
+The basic procedures for getting monitoring data are simple and unified as below.
+
+At first, set up the environment for resources and attributes that are monitored:
+
+1. Create a resource instance with the corresponding [resource type](#resource_types).
+
+2. If required, set [resource control](#resource_controls) to specify the target among multi-instantiable resources.
+
+3. Set interesting [attributes](#resource_attributes) for created resource instance.
+
+Now, it is ready to get monitoring data. Repeat the next two steps when the application wants to get resource attributes:
+
+4. Request to update the individual resource or all resources at once.
+
+5. Get the attribute value along with the type of attributes using APIs below:
```

Comment:
This part is better represented by a nested list.
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#nested-list-spacing
```suggestion
The basic procedures for getting monitoring data are simple and unified as below:

-   At first, set up the environment for resources and attributes that are monitored:

    1.  Create a resource instance with the corresponding [resource type](#resource_types).

    2.  If required, set [resource control](#resource_controls) to specify the target among multi-instantiable resources.

    3.  Set interesting [attributes](#resource_attributes) for created resource instance.

-   Now, it is ready to get monitoring data. Repeat the next two steps when the application wants to get resource attributes:

    1.  Request to update the individual resource or all resources at once.

    2.  Get the attribute value along with the type of attributes using APIs listed in the table below.
```